SpringBootAssignment/SpringBootAssignment/src/main/java/com/Assignment/SpringBootAssignment/controller/MovieController.java

```diff
@@ -0,0 +1,53 @@
+package com.Assignment.SpringBootAssignment.controller;
+
+import com.Assignment.SpringBootAssignment.entity.Movie;
+import com.Assignment.SpringBootAssignment.services.MovieServices;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/movies")
+
+public class MovieController {
+
+    @Autowired
+    private MovieServices movieServices;
+
+    public MovieController(MovieServices movieService) {
+        movieServices=movieService;
+    }
+
+
+
+   @GetMapping("/{id}")
+    public Movie movieName(@PathVariable int id){
+        return movieServices.movieName(id);
+    }
+
+    @GetMapping("/list")
+    public List<Movie > moviesName(){
+        return movieServices.moviesName();
```

Comment:
Rename the method please.
Also can we add sorting and pagination in getAll apis ??